### PR TITLE
added run.sh

### PR DESCRIPTION
**Pull request by patrickcarver**

### CODE REVIEW

File: elixir-patrick/run.sh [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
```
Comment (drewpearce) [inline elixir-patrick/run.sh]: rename to run_test.sh

**Review by drewpearce — CHANGES_REQUESTED** [review]
need a build_test.sh if building is required.
contribute to the Dockerfile as well.

File: elixir-patrick/run.sh (hunk 1 above)
Comment (patrickcarver) [reply elixir-patrick/run.sh]: updated to run_test.sh

**Comment by patrickcarver** [comment]
Made requested changes

**Pull request merged**